test-apps/display-test-app/vite.config.ts

```diff
@@ -65,6 +65,7 @@ export default defineConfig(() => {
           /core\/electron/, // prevent error in ElectronApp
           /core\/mobile/, // prevent error in MobileApp
           /node_modules/, // prevent errors from dependencies
+          /core\/frontend/, // prevents multiple instances loaded error
```

Comment:
where is this coming from?
i didnt see this when i made changes the other day that couldnt have been resolved from L156